js/everything.js

```diff
@@ -193,6 +193,9 @@ var ZenPlayer = {
                 if (window.sessionStorage && window.sessionStorage.hasOwnProperty(videoID)) {
                     var resumeTime = window.sessionStorage[videoID];
                     var videoDuration = plyrPlayer.plyr.embed.getDuration();
+                    if (videoDuration === 0) {
```

Comment:
Here is some console output whithout fix:
<img width="1237" alt="screenshot 2017-07-24 07 48 39" src="https://user-images.githubusercontent.com/10586997/28508667-03b9de48-7045-11e7-933e-6c6581d845b5.png">
You can see that videoDuration equals 0 when some first function calls. So the closure 
```javascript
if (currentTime < videoDuration - 3) {
  resumeTime = currentTime;
}
```
has no chance to set resumeTime to anything but the default zero.